### PR TITLE
Enable testing for alpha flutter web app.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This CL enables the button to launch the flutter web version of DevTools

@DaveShuckerow I'm currently getting a not found error when clicking the button. Looks like flutter.html is not checked in. This is a release blocker: https://github.com/flutter/devtools/issues/1457

### CODE REVIEW

File: packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart [hunk 1]
```diff
@@ -241,7 +242,7 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         inspectorService: inspectorService,
         treeType: FlutterTreeType.widget,
         onExpandCollapseSupported: _onExpandCollapseSupported,
-        onLayoutExplorerSupported: _onLayoutExplorerSupported,
+        onLayoutExplorerSupported: kIsWeb ? null : _onLayoutExplorerSupported,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart]: it is better to put these sorts of tweaks in their own CL so that they can be undone with a trivial revert.

**Comment by DaveShuckerow** [comment]
> @DaveShuckerow I'm currently getting a not found error when clicking the button. Looks like flutter.html is not checked in. This is a release blocker: #1457

tool/build_release.sh should bundle the flutter build output with the pub build_runner output.  From there you can serve the app from packages/devtools

dart bin/devtools.dart
